attestation_demo/AppInitiator/UntrustedEnclaveMessageExchange.h

```diff
@@ -78,6 +78,12 @@ uint32_t send_request_ocall(uint32_t session_id, secure_message_t* req_message,
  * */
 uint32_t end_session_ocall(uint32_t session_id);
 
+/* Function Description: this is OCALL interface for initiator enclave to print log messages
+ * Parameter Description:
+ *      [input] msg: this is pointer to log message string
+ * */
+void log_ocall(const char *msg);
```

Comment:
This pull request adds a `log_ocall` function to the enclave for the standalone attestation client in order to print out the response it receives from Pelz to show that the communication and Pelz's request handling are working correctly.

Reply:
I didn't think it was necessary to declare ecalls and ocalls outside the ```.edl``` file. When the sgx edger8r runs it creates headers that should have the correct declarations. But it might be that the build flow for the demo is slightly different than for main pelz so they need to be separately declared here.

Reply:
It looks like these declarations are only necessary because the function definitions are in C++ (UntrustedEnclaveMessageExchange.cpp), and they wouldn't be linkable if these `extern "C"` declarations weren't present.
The `extern "C"` specification could be moved into UntrustedEnclaveMessageExchange.cpp, which would make these declarations redundant, but the way it is now seems fine to me.

Reply:
Thanks for clearing that up for me.